mail/mlmmj/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           codeberg 1.0
+PortGroup           legacysupport 1.1
+
+codeberg.setup      mlmmj mlmmj RELEASE_1_4_6
+
+name                mlmmj
+version             1.4.6
+categories          mail
+license             MIT
+maintainers         @inucat \
+                    proton.me:inucat
+description         A simple and slim mailing list manager
+long_description    Mlmmj is a simple and slim mailing list manager (MLM) \
+                    inspired by ezmlm. It works with many different Mail \
+                    Transport Agents (MTAs) and is simple for a system \
+                    adminstrator to install, configure and integrate with \
+                    other software. As it uses very few resources, and \
+                    requires no daemons, it is ideal for installation on \
+                    systems where resources are limited, such as Virtual \
+                    Private Servers (VPSes).
+
+checksums           rmd160  1f462719550aa4cb2410718d30979ebb99c2bb4f \
+                    sha256  5cd877703c80ab07f9ae845b89a34c032c31d290f4a311f6c8b9fe52ca508822 \
+                    size    281169
+patchfiles          include-xlocale_h.diff \
+                    remove-disabled-test.diff
+use_autoreconf      yes
+configure.args       --disable-tests
+configure.env-append LIBS=-liconv
```

We have a dedicated variable for this:
```suggestion
configure.libs       -liconv
```
It would also be good to add the upstream bug report URL in a comment above this line so that we can remove this when the bug is fixed.

---

You're asking to link with libiconv but you haven't declared that you depend on the libiconv port. Do that by adding:
```
depends_lib port:libiconv
```

---

I wasn’t meant perhaps, but `libiconv` also sits in a system prefix, I think.

---

I confirmed two patterns:

- When there is only macOS libiconv, it is used.

```
% otool -L src/mlmmj-maintd
src/mlmmj-maintd:
	/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1281.100.1)
```

- However, if libiconv is installed by MacPorts, the MacPorts one will be used.

---

@inucat Then just declare a dependency, IMO. MacPorts one is preferred, since that will ensure predictable builds.

---

Thank you, the dependency `depends_lib port:libiconv` now has been declared.